src/app/static/src/app/store/root/mutations.ts

```diff
@@ -158,3 +164,11 @@ export const mutations: MutationTree<RootState> = {
     ...languageMutations
 
 };
+
+const stopPolling = (state: any) => {
+    if (state.statusPollId == -1) {
+        return
+    }
+    clearInterval(state.statusPollId);
+    state.statusPollId = -1;
```

Comment:
There is no need for resetting poll id,  it will get reset when initialState gets applied below.

Reply:
mmh I see this is duplicated in this instance. I left this around because was thinking it would make the `stopPolling` function more reusable and matches the impl of `stopPolling` function in other mutations. I feel like this is the place to set it to `-1` as it keeps all the `stopPolling` logic in one place and can be re-used in the future.